maat-scheduled-tasks/src/main/java/uk/gov/justice/laa/maat/scheduled/tasks/service/XhibitDataService.java

```diff
@@ -96,23 +104,23 @@ public GetRecordSheetsResponse getRecordSheets(RecordSheetType recordSheetType,
         }
     }
 
-    public void markRecordsSheetsAsProcessed(List<XhibitRecordSheetDTO> recordSheets, RecordSheetType recordSheetType) {
-        renameRecordSheets(recordSheets, recordSheetType, RecordSheetStatus.PROCESSED);
+    public void markRecordSheetsAsProcessed(List<XhibitRecordSheetDTO> recordSheets, RecordSheetType recordSheetType) {
+        List<String> fileNames = recordSheets.stream().map(XhibitRecordSheetDTO::getFilename).toList();
+        renameRecordSheets(fileNames, recordSheetType, RecordSheetStatus.PROCESSED);
     }
 
     public void markRecordSheetsAsErrored(List<XhibitRecordSheetDTO> recordSheets, RecordSheetType recordSheetType) {
-        renameRecordSheets(recordSheets, recordSheetType, RecordSheetStatus.ERRORED);
+        List<String> fileNames = recordSheets.stream().map(XhibitRecordSheetDTO::getFilename).toList();
+        renameRecordSheets(fileNames, recordSheetType, RecordSheetStatus.ERRORED);
     }
 
-    private void renameRecordSheets(List<XhibitRecordSheetDTO> recordSheets,
+    private void renameRecordSheets(List<String> fileNames,
```

Comment:
Minor: can we make this `filenames` instead please? 🙂  (and by extension `fileName` defined on line 123)